pages/common/sbcl.md

```diff
@@ -0,0 +1,12 @@
+# sbcl
+
+> High performance Common Lisp compiler.
+> More information: <http://www.sbcl.org/>.
+
+- Open an SBCL interactive shell (REPL):
```

Comment:
One quick note: I like the verb 'start' for this sort of thing—thoughts?
```suggestion
- Start an SBCL interactive shell (REPL):
```

Reply:
Agreed :+1: